docs/reference/api/resources/sources/list_source.rst

```diff
@@ -42,6 +42,8 @@ the operations you'd expect on a normal Python list, such as ``insert``, ``remov
     # Insert a new row at the start of the data
     source.insert(0, name="Bettong", weight=1.2)
 
+.. _listsource-item:
+
 When initially constructing the ListSource, or when assigning a specific item in
 the ListSource, each item can be:
```

Comment:
There were four separate specifications of this; I've centralized them all here.